x-pack/plugins/security_solution/public/management/pages/policy/view/policy_forms/components/policy_form_layout.tsx

```diff
@@ -61,12 +64,23 @@ export const PolicyFormLayout = React.memo(() => {
   const [showConfirm, setShowConfirm] = useState<boolean>(false);
   const [routeState, setRouteState] = useState<PolicyDetailsRouteState>();
   const policyName = policyItem?.name ?? '';
-  const hostListRouterPath = getEndpointListPath({ name: 'endpointList' });
+  const isPolicyListEnabled = useIsExperimentalFeatureEnabled('policyListEnabled');
```

Comment:
Why do we need to check this?

Reply:
Unfortunately this feature flag is still there, so we need it to determine where the user will be redirected to when they click the `cancel` button. If this feature is `on` then the default should be policy list page. Else, it should be the Endpoint list (that is what it was before). 

I'm hoping that we can clean up several FF in this 8.7 release